be/src/exec/vectorized/hash_join_node.cpp

```diff
@@ -406,6 +407,24 @@ Status HashJoinNode::close(RuntimeState* state) {
     return ExecNode::close(state);
 }
 
+bool _is_left_join(TJoinOp::type join_type) {
```

Comment:
left_join means left outer join, it is confusing, so suggest to put follow code into a function named need_accumulate_chunk() then invoke this function.

```
    bool need_accumulate_chunk = !_is_left_join(_join_type) || !_conjunct_ctxs.empty() ||
                                 !_other_join_conjunct_ctxs.empty() || !local_rf_waiting_set().empty() ||
                                 (global_rf_collector != nullptr && !global_rf_collector->descriptors().empty());
```